packages/preview-middleware-client/src/flp/bootstrap.js

```diff
@@ -15,10 +15,24 @@ async function ushellBootstrap(fnCallback) {
         const response = await fetch(`${basePath}/resources/sap-ui-version.json`);
         const json = await response.json();
         const version = json?.libraries?.find((lib) => lib.name === 'sap.ui.core')?.version ?? '1.121.0';
-        const majorUi5Version = parseInt(version.split('.')[0], 10);
+        const [major, minor] = version.split('.');
+        const majorUi5Version = parseInt(major, 10);
+        const minorUi5Version = parseInt(minor, 10);
         if (majorUi5Version >= 2) {
             src = `${basePath}/resources/sap/ushell/bootstrap/sandbox2.js`;
         }
+        if (majorUi5Version === 1 && minorUi5Version < 72) {
```

Comment:
Please add a concise comment here explaining why you are doing what you are doing.

Reply:
Added in: e31fc3ee49d0c18c72c8e7d1a9a250e465318b47